cmd/virt-handler/virt-handler.go

```diff
@@ -496,6 +504,22 @@ func (app *virtHandlerApp) shouldChangeRateLimiter() {
 	log.Log.V(2).Infof("setting rate limiter to %v QPS and %v Burst", qps, burst)
 }
 
+// Install the SELinux policy when the feature gate that disables it gets removed
+func (app *virtHandlerApp) shouldInstallSELinuxPolicy() {
+	app.semoduleLock.Lock()
+	defer app.semoduleLock.Unlock()
+	if app.customSELinuxPolicyInstalled {
+		return
+	}
+	if !app.clusterConfig.CustomSELinuxPolicyDisabled() {
+		se, exists, err := selinux.NewSELinux()
+		if err == nil && exists {
+			app.installCustomSELinuxPolicy(se)
+			app.customSELinuxPolicyInstalled = true
+		}
+	}
```

Comment:
How this feature gate is supposed to be used? I don't see any code that 'uninstalls' the SELinux policy here. If I deploy KubeVirt with the default settings, it will install the custom policy. Then my assumption would be that if I edit the CR and add the feature gate, then it should be disabled, though this seems not the case now. Maybe I am missing smth?

Does that mean that the feature gate only takes efect if it is specified in the KubeVirt CR prior to the deployment?

Reply:
> How this feature gate is supposed to be used? I don't see any code that 'uninstalls' the SELinux policy here. If I deploy KubeVirt with the default settings, it will install the custom policy. Then my assumption would be that if I edit the CR and add the feature gate, then it should be disabled, though this seems not the case now. Maybe I am missing smth?

The feature gate is used by the cluster admin to inform us that their container-selinux policy is recent-enough for all KubeVirt use-cases (i.e. >= v2.193.0).
If set at install time, the policy will not be installed. If set later, the policy will just stop being used, but won't be uninstalled, as you noticed.
However, the code is written in such a way that we can easily add an uninstall. I just thought it'd be best to avoid messing with the node's policy any further, but I'm open to the idea of uninstalling it every time the feature gate is enabled if desired.
My preference would be to add a line to the docs, stating that the policy doesn't auto-uninstall but that it can be done manually if needed, with `semodule -r virt_launcher`.

> 
> Does that mean that the feature gate only takes efect if it is specified in the KubeVirt CR prior to the **first** deployment?

The feature gate always takes its effect of disabling the **use** of the policy (see changes in `pkg/virt-controller/services/template.go`)
Additionally, if it is specified prior to the first deployment, you get the benefit of it not even getting installed.

Reply:
Ah, okay, I see now. Thanks for clarifying it.

> My preference would be to add a line to the docs, stating that the policy doesn't auto-uninstall

I think that would be useful.

Reply:
Seems I got confused by `disable` != `uninstall` here :)

Reply:
> > My preference would be to add a line to the docs, stating that the policy doesn't auto-uninstall
> 
> I think that would be useful.

Done, PTAL!
@xpivarc ok to unhold?

Reply:
Thx!